application/console/controllers/CronController.php

```diff
@@ -14,4 +18,61 @@ public function actionRemoveOldUnverifiedRecords()
         ]);
         Mfa::removeOldUnverifiedRecords();
     }
+
+    /**
+     * Send events to Google Analytics that give the number of ...
+     *  - active users
+     *  - active users that have a verified Mfa of any type
+     *  - active users with a backup code Mfa
+     *  - active users with a verified totp Mfa
+     *  - active users with a verified u2f Mfa
+     *
+     * @throws \Exception
+     */
+    public function actionGoogleAnalytics()
+    {
+        $trackingId = Env::get('GA_TRACKING_ID', null); // 'UA-12345678-12'
```

Comment:
Shouldn't this be in `common/config/main.php` as part of `params`, perhaps a `googleAnalytics` element with `trackingId` and `clientId` children? I believe all our other `Env` usage is in the config for central view and usage.